modules/swagger-codegen/src/main/resources/go/configuration.mustache

```diff
@@ -27,7 +28,8 @@ func NewConfiguration() *Configuration {
         Debug: false,
         DefaultHeader: make(map[string]string),
         ApiKey: make(map[string]string),
-        ApiKeyPrefix: make(map[string]string) ,
+        ApiKeyPrefix: make(map[string]string),
+        UserAgent: "{{#httpUserAgent}}{{{.}}}{{/httpUserAgent}}{{^httpUserAgent}}Swagger-Codegen/{{{version}}}/go{{/httpUserAgent}}",
```

Comment:
@guohuang I think {{version}} (the API spec version) should be replaced by {{packageVersion}} (SDK version)